src/expr/type_node.cpp

```diff
@@ -459,9 +459,16 @@ TypeNode TypeNode::instantiateParametricDatatype(
   return nm->mkTypeNode(kind::PARAMETRIC_DATATYPE, paramsNodes);
 }
 
+unsigned TypeNode::getSortConstructorArity() const
```

Comment:
```suggestion
uint64_t TypeNode::getSortConstructorArity() const
```

The `SortArityAttr` atrribute maps to `uint64_t`. We could also consider returning a `size_t` here, I have no strong opinions.